vm.tf

```diff
@@ -38,6 +38,11 @@ variable "vision_tar_name" {
   default = "visual-insights-images-1.2.0.0.tar"
 }
 
+variable "example_dataset_name" {
+  description = "Name of example dataset to automatically import into Visual Insights."
```

Comment:
I'm torn here; should this be relative to our bucket base, or should it be an absolute URL? On the one hand, having an absolute URL will be super-handy for ppl who want to set up some sort of class; i.e. they'll be able to point to some random COS bucket or URL and say "here, use this," and it'll get prepopulated in. On the other hand, that now means we'll have two similar-ish but different ways to handle injecting assets into the the VM. What do you think about changing this to a fully-specified URL, ala "https://foo/bar/baz.zip" ?

Reply:
I think it would work out to change it to a fully-specific URL! Was mainly basing it just on the other variables in the schematic, but would also make sense to just have it a full URL as well and for other people to easily specify their own file.